sp800-63b/sec5_authenticators.md

```diff
@@ -314,21 +316,19 @@ CSPs SHALL provide subscriber instructions on how to appropriately  protect the
 
 #### <a name="throttle"></a>5.2.2. Rate Limiting (Throttling)
 
-*cf. 800-63-2 sec 8.2.3, p.75*
-
-When the authenticator output or activation secret does not have sufficient entropy, the verifier SHALL implement controls to protect against online guessing attacks. Unless otherwise specified in the description of a given authenticator, the verifier SHALL effectively limit online attackers to 100 consecutive failed attempts on a single account in any 30 day period.
+When the authenticator output or activation secret does not have sufficient entropy, the verifier SHALL implement controls to protect against online guessing attacks. Unless otherwise specified in the description of a given authenticator, the verifier SHALL effectively limit online attackers to 100 consecutive failed attempts on a single account.
 
 Additional techniques MAY be used to prioritize authentication attempts that are likely to come from the subscriber over those that are more likely to come from an attacker:
 
 - Requiring the claimant to complete a Completely Automated Public Turing test to tell Computers and Humans Apart (CAPTCHA) before attempting authentication
 
-- Requiring the claimant to wait for a short period of time (anything from 30 seconds to an hour, depending on how close the system is to its maximum allowance for failed attempts) before attempting Authentication following a failed attempt
+- Requiring the claimant to wait following a failed attempt for a period of time that is increasing in intervals from, say, 30 seconds to an hour, as the account approaches its maximum allowance for consecutive failed attempts
 
 - Only accepting authentication requests from a white list of IP addresses at which the subscriber has been successfully authenticated before
 
 - Leveraging other risk-based or adaptive authentication techniques to identify user behavior that falls within, or out of, typical norms.
 
-Since these measures often create user inconvenience, the verifier SHOULD allow a certain number of failed authentication attempts before employing the above techniques.
+Since these measures often create user inconvenience, the verifier SHOULD allow a several failed authentication attempts before employing the above techniques.
```

Comment:
I think we should delete this requirement.  Make this a UX consideration. If service wants to lock out after 1 attempt, so be it.  'Since these measures often create user inconvenience, the verifier SHOULD allow a several failed authentication attempts before employing the above techniques.'